WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -25,11 +27,38 @@ final class NewOrderViewModel: ObservableObject {
     ///
     @Published var presentNotice: NewOrderNotice?
 
-    init(siteID: Int64, stores: StoresManager = ServiceLocator.stores) {
+    @Published var orderStatus: OrderStatus
```

Comment:
Do you think it is possible to not expose a `Networking/Yosemite` model to the view? It kinda breaks the separation of concerns 😅